XRTK-Core/Packages/com.xrtk.core/Inspectors/Profiles/MixedRealityServiceProviderProfileInspector.cs

```diff
@@ -206,6 +206,18 @@ private void DrawConfigurationOptionElement(Rect rect, int index, bool isActive,
                 }
             }
 
+            if (configurationProfileProperty.objectReferenceValue != null)
```

Comment:
I need to double check this change later with some other branches.

Reply:
Yeah this change should have made it into the dev branch already. 

I will check to make sure everything is synced up correctly

Reply:
Ah I see, it's in here twice
https://github.com/XRTK/XRTK-Core/pull/216/files/227dace594a74ffe3f8b981bcb5edff5853f6d8b#diff-057d09042039ea5ac264e1da08fc4d4cL197